src/SDFExtension.cc

```diff
@@ -74,6 +75,7 @@ SDFExtension::SDFExtension(const SDFExtension &_ge)
   this->material = _ge.material;
   this->visual_blobs = _ge.visual_blobs;
   this->collision_blobs = _ge.collision_blobs;
+  this->isSetStaticFlag = _ge.isSetStaticFlag;
```

Comment:
Can we do something about this line in codecov ?

Reply:
I just removed the SDFExtension copy constructor and destructor so the class can follow the rule of zero. That should improve coverage.

Reply:
coverage checker is green; removed 45 missed lines